'BuildMeta' object has no attribute 'add_public_includes'

I built `cumm` from source and tried to import `cumm` but failed:

> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\resources\cumm\cumm\__init__.py", line 33, in <module>
    verbose=False)
  File "C:\Users\Yihua\AppData\Roaming\Python\Python37\site-packages\pccm\builder\pybind.py", line 56, in build_pybind
    user_cus = cg.build_graph(cus, namespace_root)
  File "C:\Users\Yihua\AppData\Roaming\Python\Python37\site-packages\pccm\core\__init__.py", line 1903, in build_graph
    cu_type_to_cu[dep] = dep()
  File "C:\Users\Yihua\AppData\Roaming\Python\Python37\site-packages\pccm\core\__init__.py", line 844, in wrapper
    func(self, *args, **kwargs)
  File "d:\resources\cumm\cumm\common.py", line 278, in __init__
    self.build_meta.add_public_includes(TENSORVIEW_INCLUDE_PATH)
AttributeError: 'BuildMeta' object has no attribute 'add_public_includes'

Environment:
Windows 11
Python 3.7.13
pccm==0.3.4
ccimport==0.3.7
pip==21.2.4
mkl-service==2.4.0

I noticed that there were recent changes in `ccimport` so I tried another environment and built `ccimport` from source but also failed:

> [1/38] [MSVC][c++/pch]D:\Resources\cumm\cumm\build\core_cc\include\csrc\arrayref\ArrayPtr.h.pch|D:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\csrc\arrayref\ArrayPtr.h.cc.o
FAILED: D:/Resources/cumm/cumm/build/core_cc/msvc_stub/include/csrc/arrayref/ArrayPtr.h.cc.o
cl /I "D:\Resources\cumm\cumm\build\core_cc\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\pybind11\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\Include" /O2 /DNOMINMAX /std:c++14 /MD /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /EHsc /Zc:__cplusplus /nologo /showIncludes /DTV_CUDA /I "D:\Resources\cumm\include" /I "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.6\include" -c /Yccsrc/arrayref/ArrayPtr.h /FpD:\Resources\cumm\cumm\build\core_cc\include\csrc\arrayref\ArrayPtr.h.pch D:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\csrc\arrayref\ArrayPtr.h.cc /FoD:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\csrc\arrayref\ArrayPtr.h.cc.o
D:\Resources\cumm\include\tensorview\core\array.h(813): warning C4068: Unknown pragma“GCC”
D:\Resources\cumm\include\tensorview/core/common.h(29): fatal error C1083: Cannot open include file: “cuda.h”: No such file or directory
[2/38] [MSVC][c++/pch]D:\Resources\cumm\cumm\build\core_cc\include\tensorview_bind\TensorViewBind.h.pch|D:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\tensorview_bind\TensorViewBind.h.cc.o
FAILED: D:/Resources/cumm/cumm/build/core_cc/msvc_stub/include/tensorview_bind/TensorViewBind.h.cc.o
cl /I "D:\Resources\cumm\cumm\build\core_cc\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\pybind11\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\Include" /O2 /DNOMINMAX /std:c++14 /MD /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /EHsc /Zc:__cplusplus /nologo /showIncludes /DTV_CUDA /I "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.6\include" /I "D:\Resources\cumm\include" -c /Yctensorview_bind/TensorViewBind.h /FpD:\Resources\cumm\cumm\build\core_cc\include\tensorview_bind\TensorViewBind.h.pch D:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\tensorview_bind\TensorViewBind.h.cc /FoD:\Resources\cumm\cumm\build\core_cc\msvc_stub\include\tensorview_bind\TensorViewBind.h.cc.o
D:\Resources\cumm\include\tensorview/core/common.h(29): fatal error C1083: Cannot open include file: “cuda.h”: No such file or directory
[3/38] [MSVC][c++]D:\Resources\cumm\cumm\build\core_cc\src\tensorview_bind\PyBindTensorViewBind\PyBindTensorViewBind_bind_TensorViewBind.cc.o
FAILED: D:/Resources/cumm/cumm/build/core_cc/src/tensorview_bind/PyBindTensorViewBind/PyBindTensorViewBind_bind_TensorViewBind.cc.o
cl /I "D:\Resources\cumm\cumm\build\core_cc\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\pybind11\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\Include" /O2 /DNOMINMAX /std:c++14 /MD /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /EHsc /Zc:__cplusplus /nologo /showIncludes /DTV_CUDA /I "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.6\include" /I "D:\Resources\cumm\include" -c D:\Resources\cumm\cumm\build\core_cc\src\tensorview_bind\PyBindTensorViewBind\PyBindTensorViewBind_bind_TensorViewBind.cc /FoD:\Resources\cumm\cumm\build\core_cc\src\tensorview_bind\PyBindTensorViewBind\PyBindTensorViewBind_bind_TensorViewBind.cc.o
D:\Resources\cumm\include\tensorview/core/common.h(29): fatal error C1083: Cannot open include file: “cuda.h”: No such file or directory
[4/38] [MSVC][c++]D:\Resources\cumm\cumm\build\core_cc\src\csrc\arrayref\PyBindArrayPtr\PyBindArrayPtr_bind_ArrayPtr.cc.o
FAILED: D:/Resources/cumm/cumm/build/core_cc/src/csrc/arrayref/PyBindArrayPtr/PyBindArrayPtr_bind_ArrayPtr.cc.o
cl /I "D:\Resources\cumm\cumm\build\core_cc\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\pybind11\include" /I "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\Include" /O2 /DNOMINMAX /std:c++14 /MD /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /EHsc /Zc:__cplusplus /nologo /showIncludes /DTV_CUDA /I "D:\Resources\cumm\include" /I "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.6\include" -c D:\Resources\cumm\cumm\build\core_cc\src\csrc\arrayref\PyBindArrayPtr\PyBindArrayPtr_bind_ArrayPtr.cc /FoD:\Resources\cumm\cumm\build\core_cc\src\csrc\arrayref\PyBindArrayPtr\PyBindArrayPtr_bind_ArrayPtr.cc.o
D:\Resources\cumm\include\tensorview\core\array.h(813): warning C4068: Unknown pragma“GCC”
D:\Resources\cumm\include\tensorview/core/common.h(29): fatal error C1083: Cannot open include file: “cuda.h”: No such file or directory
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\resources\cumm\cumm\__init__.py", line 29, in <module>
    pccm.builder.build_pybind([ArrayPtr(), TensorViewBind()],
  File "D:\Resources\pccm\pccm\builder\pybind.py", line 114, in build_pybind
    return ccimport.ccimport(
  File "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\ccimport-0.4.0-py3.9.egg\ccimport\core.py", line 149, in ccimport
  File "F:\Program Files (x86)\Intel\oneAPI\intelpython\latest\lib\site-packages\ccimport-0.4.0-py3.9.egg\ccimport\buildtools\writer.py", line 990, in build_simple_ninja
subprocess.CalledProcessError: Command '['ninja']' returned non-zero exit status 1.

Environment:
Windows 11
Python 3.9.7
pccm==0.3.5 (built from source)
ccimport==0.4.0 (built from source)
pip==21.2.4

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ The c++ code will be built automatically when you change c++ code in project.
 
 0. uninstall cumm installed by pip. you must ensure no "cumm" exists in ```pip list | grep cumm```
 1. install build-essential, install CUDA
-2. ```git clone https://github.com/FindDefinition/cumm```, ```cd ./cumm```, ```pip install -e .```
+2. ```git clone https://github.com/FindDefinition/cumm```, ```cd ./cumm```, ```git checkout tags/<tag_name>```, ```pip install -e .```
 3. in python, ```import cumm``` and wait for build finish.
 
 #### Windows
@@ -49,7 +49,7 @@ The c++ code will be built automatically when you change c++ code in project.
 1. install visual studio 2019 or newer. make sure C++ development component is installed. install CUDA
 2. set [powershell script execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.1)
 3. start a new powershell, run ```tools/msvc_setup.ps1```
-4. ```git clone https://github.com/FindDefinition/cumm```, ```cd ./cumm```, ```pip install -e .```
+4. ```git clone https://github.com/FindDefinition/cumm```, ```cd ./cumm```, ```git checkout tags/<tag_name>```, ```pip install -e .```
 5. in python, ```import cumm``` and wait for build finish.
 
 ### Build wheel from source 

